### PR TITLE
Remove font-size mixin

### PR DESCRIPTION
**Pull request by neilabraham**
#### Changes proposed in this Pull Request:

Remove the font-size mixin now that we no longer need pixel fallback support for IE8 and below.

#### Related issue(s):

### CODE REVIEW

File: sass/forms/_buttons.scss [hunk 1]
```diff
@@ -8,7 +8,7 @@ input[type="submit"] {
 	background: $color__background-button;
 	color: rgba(0, 0, 0, 0.8);
```
Comment (Ismail-elkorchi) [inline sass/forms/_buttons.scss:10]:
```suggestion
```

File: sass/forms/_buttons.scss [hunk 2]
```diff
@@ -8,7 +8,7 @@ input[type="submit"] {
...
 
-	@include font-size(0.75);
+	font-size: 0.75rem;
```
Comment (Ismail-elkorchi) [inline sass/forms/_buttons.scss:12]:
```suggestion
```

File: sass/modules/_accessibility.scss [hunk 3]
```diff
@@ -20,7 +20,7 @@
 		color: $color__text-screen;
 		display: block;
```
Comment (Ismail-elkorchi) [inline sass/modules/_accessibility.scss:22]:
```suggestion
```

File: sass/modules/_accessibility.scss [hunk 4]
```diff
@@ -20,7 +20,7 @@
...
 
-		@include font-size(0.875);
+		font-size: 0.875rem;
```
Comment (Ismail-elkorchi) [inline sass/modules/_accessibility.scss:24]:
```suggestion
```

File: sass/typography/_copy.scss [hunk 5]
```diff
@@ -21,7 +21,7 @@ pre {
 	background: $color__background-pre;
 	font-family: $font__pre;
```
Comment (Ismail-elkorchi) [inline sass/typography/_copy.scss:23]:
```suggestion
```

File: sass/typography/_copy.scss [hunk 6]
```diff
@@ -35,7 +35,7 @@ tt,
 var {
 	font-family: $font__code;
```
Comment (Ismail-elkorchi) [inline sass/typography/_copy.scss:37]:
```suggestion
```

File: sass/typography/_typography.scss [hunk 7]
```diff
@@ -7,7 +7,7 @@ textarea {
 	color: $color__text-main;
 	font-family: $font__main;
```
Comment (Ismail-elkorchi) [inline sass/typography/_typography.scss:9]:
```suggestion
```

**Comment by neilabraham** [comment]
Of course.  Sorry I missed the empty lines and forgot to include the CSS in the first commit.

**Pull request merged**